docs/current/cookbook/snippets/cache-dependencies/index.mjs

```diff
@@ -0,0 +1,24 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(
+  async (client) => {
+    // use a node:18 container
+    // mount the source code directory on the host
+    // at /src in the container
+    // mount the cache volume to persist dependencies
+    const source = client
+      .container()
+      .from("node:18")
+      .withDirectory("/src", client.host().directory("."))
+      .withWorkdir("/src")
+      .withMountedCache(
+        "/src/node_modules",
+        client.cacheVolume("node-18-myapp-myenv")
+      )
```

Comment:
The equivalent to the others would be the npm cache, which can be added in addition to this one (with a more global reuse, by node version).

Reply:
How should this be added?